allennlp/training/regularizer.py

```diff
@@ -0,0 +1,14 @@
+import torch
+
+
+class Regularizer:
+    """
+    An abstract class representing a regularizer. It must implement
+    call, returning a scalar tensor.
+    """
+
+    def __init__(self, **kwargs):
```

Comment:
Joel and I decided this was the lesser of too evils - without this, Mypy complains that `Regularizers` have too many arguments when instantiating them [here](https://github.com/allenai/allennlp/pull/21/files#diff-704f6590671715f995ff9148743a01b0R114)